modules/cnf-migrating-from-amqp-to-http-transport.adoc

```diff
@@ -0,0 +1,184 @@
+// Module included in the following assemblies:
+//
+// * networking/using-ptp.adoc
+
+:_content-type: PROCEDURE
+[id="cnf-migrating-from-amqp-to-http-transport_{context}"]
+= Migrating consumer applications to use HTTP transport for PTP and bare-metal events
+
+If you have previously deployed PTP or bare-metal events consumer applications, you need to update the applications to use HTTP message transport.
+
+You need to configure persistent storage for message subscriptions, set the `transportHost` in the PTP or {redfish-operator} configuration, and update existing consumer applications to use the new HTTP `transportHost`.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+
+* You have logged in as a user with `cluster-admin` privileges.
+
+* You have updated the PTP Operator or {redfish-operator} to version 4.13+ which uses HTTP transport by default.
+
+.Procedure
+
+. Configure persistent storage to persist events subscriptions in the pod managed by the PTP Operator or {redfish-operator}.
+For example, apply the following `PersistentVolumeClaim` (PVC) custom resource (CR) in the cluster:
```

Comment:
This step is done by the ptp/BMER operator when applying the `PtpOperatorConfig` or `HardwareEvent` CRs.